src/components/DatePicker/CalendarHeader/__tests__/CalendarHeader.spec.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
+import { CalendarHeader } from '../CalendarHeader';
+
+describe('CalendarHeader', () => {
+    const YEARS = [2016, 2025] as [number, number];
+    const decreaseMonth = jest.fn();
+    const increaseMonth = jest.fn();
+    const changeYear = jest.fn();
+    const changeMonth = jest.fn();
+    const decreaseYear = jest.fn();
+    const increaseYear = jest.fn();
+    const currentMonth = new Date();
+    let view;
```

Comment:
My TS was complaining that view has no type defined, it is any at this moment. It should have type of RenderResult I think

Reply:
I've added, but i don't get any warnings